engine/polyglot-api/src/main/scala/org/enso/polyglot/TopScope.scala

```diff
@@ -34,7 +34,24 @@ class TopScope(private val value: Value) {
     value.invokeMember(UNREGISTER_MODULE, qualifiedName): Unit
   }
 
-  def compile(shouldCompileDependencies: Boolean): Unit = {
-    value.invokeMember(COMPILE, shouldCompileDependencies)
+  def compile(
+    shouldCompileDependencies: Boolean
+  ): Unit = {
+    compile(shouldCompileDependencies, None)
   }
+  def compile(
+    shouldCompileDependencies: Boolean,
+    generateDocs: Option[String]
+  ): Unit = {
+    val docsArg = generateDocs.map {
+      case "api" => "api"
+      case _     => "md"
+    }
+    value.invokeMember(
+      COMPILE,
+      shouldCompileDependencies,
+      docsArg.getOrElse(false)
```

Comment:
is this correct? It seems that the underlying value of `docsArg` is a `String`?

Reply:
The idea was:
- `false` - don't generate
- `true` - generate and use default "style"
- any `String` - generate and use the "style" specified by the string